optimum/commands/export/onnx.py

```diff
@@ -175,6 +175,24 @@ def parse_args_onnx(parser):
         help="Enables onnxslim optimization.",
     )
 
+    optional_group.add_argument(
+        "--use-dynamo",
```

Comment:
Is this user facing? The export team from PyTorch recommended something like `--use-pt2` to disambiguate, because `dynamo` is really a legacy name in the onnx export api and is not really what the exporter uses to capture the graph. (It uses torch.export non-strict mode and aot autograd) to get the fx graph/exported program.